utils/database.js

```diff
@@ -67,30 +66,26 @@ class Database {
   ) {
     let baniLength;
     switch (length) {
-      case "EXTRA_LONG":
-        baniLength = "existsBuddhaDal";
+      case constant.EXTRA_LONG:
+        baniLength = constant.EXISTS_BUDDHA_DAL;
         break;
-      case "LONG":
-        baniLength = "existsTaksal";
+      case constant.LONG:
+        baniLength = constant.EXISTS_TAKSAL;
         break;
-      case "MEDIUM":
-        baniLength = "existsMedium";
+      case constant.MEDIUM:
+        baniLength = constant.EXISTS_MEDIUM;
         break;
-      case "SHORT":
-        baniLength = "existsSGPC";
+      case constant.SHORT:
+        baniLength = constant.EXISTS_SGPS;
         break;
       default:
-        baniLength = "existsMedium";
+        baniLength = constant.EXISTS_MEDIUM;
```

Comment:
Prefer es6 object destructuring here veerji. You don't have to use dot notation here. Even better and ideal is named import.

Reply:
fixed